### DISCUSSION TITLE
status_code variable contains status and text, not only status_code

### DISCUSSION THREAD

**Discussion by nesies** [Kludex/uvicorn#2181] [Potential Issue] [answered]
I use a custom logformat and want to return the http status_code

But status_code contains code and text.

https://github.com/encode/uvicorn/blob/7d274ed3894b8e16fcbdc69a6456f8ca12b33711/uvicorn/logging.py#L107

```
LOGGING_CONFIG["formatters"]["access"]["fmt"] = \
        "status=%(status_code)s"
```

it displays:
````
status="404 Not Found"
```

**Comment by Kludex** [comment, marked as answer]
Correct.

**Comment by nesies** [comment]
I think, it would be convenient to change it and have 2 variables
status_code (== 404
status_message (== Not Found")
